nebula_ros/config/lidar/velodyne/VLP16.param.yaml

```diff
@@ -1,6 +1,6 @@
 /**:
   ros__parameters:
-    host_ip: 255.255.255.255
+    host_ip: 192.168.1.10
```

Comment:
Very minor point, but all Velodyne datasheets seem to use 192.168.1.222 in the example setup. Although some additionally use 192.168.1.77 to show how to set up multiple sensors, so using the same as the user manual isn't really going to make things easier to understand anyway...